src/main/java/org/spongepowered/common/entity/EntityUtil.java

```diff
@@ -696,11 +696,21 @@ public static Profession validateProfession(int professionId) {
                 // Vanilla behaviour - Delete the known bed location if invalid
                 bedPos = null; // null = remove location
             }
-            // Set the new bed location for the new dimension
-            int prevDim = playerIn.dimension; // Temporarily for setSpawnPoint
+
+            // First revision: Xakep_SDK, 28.01.18. MC Version: 1.12.2, API version: 7.1.0
```

Comment:
Comment formatting is somewhat awkward. I'd prefer to see the date spelled out like Overwrites with the MC version and API version being in separate lines for readability.

Reply:
This allow's revision stacking:
First revision: Who, date, mc ver., api ver.
Second revision: Who, date, mc ver., api ver.
Third revision: Who, date, mc ver., api ver.
So we can easily search commits by dates.